### PR TITLE
initialize php api client

### PR DESCRIPTION
**Pull request by solivier**

### CODE REVIEW

File: LICENCE.txt [hunk 1]
```diff
@@ -0,0 +1,8 @@
+Akeneo PIM PHP Client
+
+The Open Software License version 3.0
+
+Copyright (c) 2013-2017, Akeneo SAS.
+
+Full license is at: http://opensource.org/licenses/OSL-3.0
```
Comment (solivier) [inline LICENCE.txt:7]: not sure about the license

File: composer.json [hunk 2]
```diff
@@ -0,0 +1,31 @@
+{
+  "name": "akeneo/php-api-client-ee",
+  "description": "Akeneo PIM ENTERPRISE client for the API",
+  "license": "OSL-3.0",
```
Comment (solivier) [inline composer.json:4]: same about the license

File: composer.json [hunk 3]
```diff
@@ -0,0 +1,31 @@
...
+  "minimum-stability": "beta",
+  "authors": [
+    {
+      "name": "Akeneo",
+      "homepage": "http://www.akeneo.com"
+    }
+  ],
+  "autoload": {
+    "psr-4": {
+      "Akeneo\\Pim\\": "src/",
```
Comment (solivier) [inline composer.json]: what about the namespaces ?

Comment (ahocquard) [reply composer.json]: AkeneoEnterprise if we do like in the Pim. And it allows composition of the CE client for a given class name (should not occur on a client, but maybe one day..).

File: LICENCE.txt (hunk 1 above)
Comment (ahocquard) [reply LICENCE.txt:7]: I asked to @nidup for the CE, maybe it can tell us, because I don't know too

File: composer.json [hunk 4]
```diff
@@ -0,0 +1,31 @@
...
+      "Akeneo\\Pim\\tests\\": "tests/"
+    }
+  },
+  "require": {
+    "akeneo/api-php-client": "^1.0@beta",
+    "php-http/guzzle6-adapter": "^1.1"
```
Comment (ahocquard) [inline composer.json]: No, it's the responsibility of the developper to checkout the adapter

File: LICENCE.txt (hunk 1 above)
Comment (solivier) [reply LICENCE.txt:7]: I asked we should use OSL-3.0

**Review by skeleton — APPROVED** [review]

**Review by momoss — APPROVED** [review]

**Review by ahocquard — APPROVED** [review]

**Pull request merged**